pkgs/servers/irc/inspircd/default.nix

```diff
@@ -150,45 +151,43 @@ stdenv.mkDerivation rec {
     sha256 = "0x3paasf4ynx4ddky2nq613vyirbhfnxzkjq148k7154pz3q426s";
   };
 
-  patches = [
-    # Remove at next release, makes --prefix and --system work together
-    (fetchpatch {
-      url = "https://github.com/inspircd/inspircd/commit/b378b5087b41f72a1624ebb58990180e0b0140aa.patch";
-      sha256 = "0c0fmhjbkfh2r1cmjrm5d4whcignwsyi6kwkhmcvqy9mv99pj2ir";
-    })
-  ];
+  outputs = [ "bin" "lib" "man" "doc" "out" ];
```

Comment:
Doesn't this need `outputBin = "out"` or something?

Reply:
I never saw that attr before, usually the default is to put `bin` first and then it Just Works.

Reply:
I've seen that pattern sometimes of naming the default output `out` regardless of what is is and then doing some renaming, but it hardly matters. (I think in this case it'd even be `bin = "out"` bossibly?)